WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginUtils.java

```diff
@@ -28,7 +29,8 @@ public static boolean isPluginFeatureAvailable(SiteModel site) {
         return site.isWPCom()
                && SiteUtils.hasNonJetpackBusinessPlan(site)
                && site.getHasCapabilityManageOptions() // Automated Transfer require admin capabilities
-               && !site.getUrl().contains(".wordpress.com") // Automated Transfers require custom domains
+               // Automated Transfers require custom domains
+               && (!site.getUrl().contains(".wordpress.com") || BuildConfig.DOMAIN_REGISTRATION_ENABLED)
```

Comment:
Could we add a variable for both of these to make the code self-documenting? I think it was fine when we had a simple logic for just checking if the url doesn't have `wordpress.com` but now with the build config, it can be a little confusing. It could be something like:

```
val isWordPressComUrl = site.getUrl().contains(".wordpress.com")
val customDomainRegistrationAvailable = BuildConfig.DOMAIN_REGISTRATION_ENABLED
```